akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/ProtoAnySerialization.scala

```diff
@@ -132,19 +159,37 @@ import scalapb.options.Scalapb
         encode(msg)
       case msg: GeneratedMessageV3 =>
         encode(msg)
+      case SerializedEvent(bytes, id, manifest) =>
```

Comment:
@johanandren tomorrow, would be good if you can sanity check this file (again). I think this is working but somewhat messy to go back and forth between typeUrl and class manifest

Reply:
Worked through it more carefully now